src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,73 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
+  if (val1 === "Hello") {
```

Comment:
`val1 === 'Hello'` is a boolean statement, so you don't need the if statement here. You could just return 
```js
return val1 === "Hello"
```